### ISSUE TITLE
librarycarpentry/lc-open-refine

### ISSUE THREAD

**Issue by zkamvar** [carpentries/lesson-transition#65]
tracking issue for <https://github.com/librarycarpentry/lc-open-refine>

- [x] fix figures in assets
- [x] investigate how figures were turned into links (edit: they weren't that's just the reporting syntax)
- [x] Link to 'setup' page in pre-requisites make no sense after transition as the lesson summary and setup information is in a single page (defeer to maintainers)
- [ ] In Setup the link to Binder using the "My Binder" image with markup results in image being rendered by not linked. (will be fixed in https://github.com/carpentries/sandpaper/issues/445) 
- [ ] First table in Episode 1 wraps the dates in the second column. Would be better for them to be on a single line as there is plenty of room to do so and they will be easier to read (only in mobile view)
- [ ] Consecutive images in Episode 3 illustrating rows vs records appear with no spacing between them (solution: add space between them)

```markdown
episodes/02-importing-data.md:65 [missing file]: [](assets/fig/openrefine_ui.png)
episodes/03-working-with-data.md:54 [missing file]: [](assets/fig/rows.png)
episodes/03-working-with-data.md:55 [missing file]: [](assets/fig/records.png)
episodes/06-working-with-columns.md:37 [missing file]: [](assets/fig/sort-menu-highlight.png)
episodes/13-looking-up-data.md:71 [missing file]: [](assets/fig/openrefine_add_columns_by_url.png)
```

**Comment by ostephens** [comment]
Just making notes as I come across them, hope that's OK

* Link to 'setup' page in pre-requisites make no sense after transition as the lesson summary and setup information is in a single page

**Comment by zkamvar** [comment]
Hi @ostephens,

I apologise for my late reply. Thank you for reporting these, I've added them to my tracking list above.

WRT to setup page, I would defer to the maintainers for this. It was a concious decision to combine the setup and index pages because the learners need this information when starting the course

WRT to the binder setup link, that's a bug in sandpaper: https://github.com/carpentries/sandpaper/issues/445 and I can try to get that fixed today.

WRT to wrapped dates, this looks to be a bug in varnish in mobile view: https://github.com/carpentries/varnish/issues/80

WRT to combined images: the solution is to add a new line in between the images. This partially due to the fact that markdown treats items on adjacent lines as being part of the same paragraph. I can add the vertical space if you want.

**Comment by ostephens** [comment]
@zkamvar no problem at all. I'm happy to tackle anything that is a formatting or lesson structure issue (setup page/combined images) so feel free to just deal with those issues that directly relate to the migration process

**Comment by zkamvar** [comment]
fixed in 42c662e0c8485af51c3e939d7f26bb08769c9546
